### PR TITLE
Bump to 4.4.1 and update requirements

### PR DESCRIPTION
**Pull request by igortg**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

This changes #33 to add new requirements restriction.

I had also took the opportunity to update the list of maintainers (@edetec moved to a new job and is not working with Conda anymore. I took part of his responsibilities in the team). If this change needs a new PR, please let me know.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,77 +1,76 @@
...
+  summary: Distributed Task Queue
+  dev_url: https://github.com/celery/celery
+  doc_url: http://docs.celeryproject.org/en/latest/index.html
 
 extra:
-    recipe-maintainers:
-        - kwilcox
-        - pmlandwehr
-        - sodre
-        - edetec
```
Comment (synapticarbors) [inline recipe/meta.yaml]: Has @edetec approved being removed as a maintainer?

**Comment by igortg** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,77 +1,76 @@
...
-        - pip
-    run:
-        - python
-        - pytz
-        - billiard >=3.6.1,<4.0
-        - kombu >=4.6.7,<5.0
-        - vine >=1.3.0
-        - sqlalchemy
-        - cryptography
+  build:
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Please change `build` to `host` here.

**Comment by igortg** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (igortg) [reply recipe/meta.yaml]: I don't know how active on GitHub is he now. If there's no answer till tomorrow, I'll rever the change.

Comment (ocefpaf) [reply recipe/meta.yaml]: Ooops. Missed this one. We'll fix once we hear from @edetec.

Comment (igortg) [reply recipe/meta.yaml]: I talked to him via IM, but I think we can wait to see if he find some time to confirm here

Comment (edetec) [reply recipe/meta.yaml]: Hi guys, sorry for taking so long to answer.
That is okay for me, remove me as a maintainer.

**Pull request merged**